### DISCUSSION TITLE
Suggestions for existing features, but expanded on.

### DISCUSSION THREAD

**Discussion by XaosKoltron** [Zrips/CMI#8577] [Feature Requests]
Most of these features are part of cmi already, but not expanded on. It would help a bunch if these were expanded in near future since I've to disable some of cmi's features in order to use features from other plugins.

Since some of the features I do disable, do have some drawbacks from it being disabled which is annoying to say, but adding these features would eliminate all issues I currently having and would be a life saver to say the least.

1) Scoreboard - Custom layout with animated and placeholders. #2883 

2) Tablist - Group weights and alphabetical formatting with LuckPerms hook, and hide players that are in spectator mode to show in survival mode instead. #931 and #1902 

3) Nametag - /nick and /glow hook for the player's name above the player with prefix/suffix. (If they dont have /nick, the glow colours the player's name with the colour of the glow) #1187 and #7525

4) Vulcan - Movement warnings hook, a list showing irregular movements and manual warnings. (If they die, hook into death inventory, display tps and ping etc) #6416 (similar)

5) LiteBans - History and active punishments hook. (/info, /checkban etc) #7580 

6) GriefPrevention - /fly hook with permissions. (cmi.fly.claim, cmi.fly.unclaim, cmi.fly.admin and cmi.fly.other) #7458 (similar)

7) DiscordSRV - /staffmsg and /helpop hook and post to assigned channel, and /helpop work with bungee. #3267 and #3464 

8) Pose Animations - (/lay, /crawl and /spin) #960 

9) Ranks - (/rankup and /ranklist gui) #1795

10) Playtime Rewards - (/prewards gui and /preward claim "reward" command) #4696 

11) TempFly - Currency based timer since current timer runs out if unused. #2699

12) Multi-Economy - Create custom economies separate from one another. (/pay "playername" "amount" "currency") #3896 

13) World Management - Create custom worlds. #5818 

14) Advancements - Create custom toast advancements with text and goals. (Like Isn't it Iron Pick etc) #5109

15) asProxy! - Create custom commands to run on bungee. (Like server restarts and when server is up, it displays a message on the bungee to all servers) #2262 

16) Automatic Backups - Scheduled backups for worlds and plugins folder, blacklist certain files to prevent massive backup folder and purge after x days from the old backups. #6636 (similar)

**Comment by Angelius782** [comment]
yes it would be wonderful ... since simplescore is over and not maintain....... :(
